nxcomp/Misc.cpp

```diff
@@ -358,6 +358,13 @@ tolerancechecks=s\n\
                ory compared to the other programs, to make easier\n\
                for the user to execute the program from the shell.\n\
 \n\
+  NX_SLAVE_CMD The full path to the slave channel handler. When the\n\
+               slave channel is enabled, the agent will listen on a\n\
+               port and forward the connection to the NX_SLAVE_CMD\n\
```

Comment:
Shouldn't we replace "port" by "socket" here?  The word port may be misleading and people assume, that there will be some IP port being opened when using this NX_SLAVE_CMD feature.